Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -682,6 +678,13 @@ - (void)doStep:(NSDictionary *)step {
   }
 }
 
+auto sortDocumentViewChange = [](std::vector<DocumentViewChange> &changes) {
```

Comment:
Oh this works too. Can you just make this a "normal" function and put it into an anonymous namespace?